bindings/wasm/src/iota/iota_document.rs

```diff
@@ -276,51 +276,52 @@ impl WasmStardustDocument {
   // Signatures
   // ===========================================================================
 
-  /// Creates a signature for the given `Credential` with the specified DID Document
-  /// Verification Method.
-  #[allow(non_snake_case)]
-  #[wasm_bindgen(js_name = signCredential)]
-  pub fn sign_credential(
-    &self,
-    credential: &WasmCredential,
-    privateKey: Vec<u8>,
-    methodQuery: &UStardustDIDUrlQuery,
-    options: &WasmProofOptions,
-  ) -> Result<WasmCredential> {
-    let mut data: Credential = credential.0.clone();
-    let private_key: PrivateKey = privateKey.into();
-    let method_query: String = methodQuery.into_serde().wasm_result()?;
-    let options: ProofOptions = options.0.clone();
-
-    self
-      .0
-      .sign_data(&mut data, &private_key, &method_query, options)
-      .wasm_result()?;
-    Ok(WasmCredential::from(data))
-  }
-
-  /// Creates a signature for the given `Presentation` with the specified DID Document
-  /// Verification Method.
-  #[allow(non_snake_case)]
-  #[wasm_bindgen(js_name = signPresentation)]
-  pub fn sign_presentation(
-    &self,
-    presentation: &WasmPresentation,
-    privateKey: Vec<u8>,
-    methodQuery: &UStardustDIDUrlQuery,
-    options: &WasmProofOptions,
-  ) -> Result<WasmPresentation> {
-    let mut data: Presentation = presentation.0.clone();
-    let private_key: PrivateKey = privateKey.into();
-    let method_query: String = methodQuery.into_serde().wasm_result()?;
-    let options: ProofOptions = options.0.clone();
-
-    self
-      .0
-      .sign_data(&mut data, &private_key, &method_query, options)
-      .wasm_result()?;
-    Ok(WasmPresentation::from(data))
-  }
+  // REFACTOR-TODO: Readd after credential module is fixed.
+  // /// Creates a signature for the given `Credential` with the specified DID Document
```

Comment:
What's wrong with the credential module? The credential and presentation structs should be unaffected.

Reply:
Refactor once #1004 is merged.